config/rbac/cert_manager_controller_approver_clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+# bind the cert-manager internal approver to approve
+# awspca.cert-manager.io CertificateRequests
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cert-manager-controller-approve:awspca-cert-manager-io
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cert-manager-controller-approve:awspca-cert-manager-io
+subjects:
+- kind: ServiceAccount
+  name: cert-manager
```

Comment:
We should probably make this configurable in case somebody changes the service account in their cert-manager installation.

Reply:
Happy to add this in @jniebuhr :slightly_smiling_face: 

I am not very experienced with kustomize. What is the most canonical way of doing this?

Reply:
Sorry, my bad. I meant the rbac file in the Helm Chart. Swapped those two